modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,254 @@ It also includes enhancements to the Eventing and Search services.
+[#table_fixedissues_v602-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34507[MB-34507^]
+| *Summary*: Fixed an issue observed when attempting to modify via subdoc by reseting the datatype to XATTRs if the pre-expiry callback returns a non-empty response.
+
+| https://issues.couchbase.com/browse/MB-34329[MB-34329^]
+| *Summary*: 
+
+| https://issues.couchbase.com/browse/MB-34173[MB-34173^]
+| *Summary*: Fixed an issue where recovery failed due to invalid snapshot start (or end) on disk. 
+
+| https://issues.couchbase.com/browse/MB-33919[MB-33919^]
+| *Summary*: The tombstone `delete-time` could be in the future or in the far past, causing the tombstone to be purged immediately or to remain on the disk for potentially long time. This has been fixed and the tombstone `delete-time` is now set to `ep_real_time` at the point of deletion.
+
+| https://issues.couchbase.com/browse/MB-33918[MB-33918^]
+| *Summary*: Fixed the cursor dropping trigger issue which caused Couchbase Server to get into a livelock state due to high checkpoint memory usage.
+
+| https://issues.couchbase.com/browse/MB-33854[MB-33854^]
+| *Summary*: Memcached returned ENGINE NOT_STORED to client after trying to perform arithmetic operations and caused a race condition.
+
+| https://issues.couchbase.com/browse/MB-33773[MB-33773^]
+| *Summary*: Fixed an issue where a chain of events, active compression of an extended attribute value followed by eviction and then expiry, triggered an exception.
+
+| https://issues.couchbase.com/browse/MB-33750[MB-33750^]
+| *Summary*: Fixed a race condition that was exposed when a supervisor that is shutting down normally, is for some reason brutally shutdown leaving an orphan child process. This rogue lease acquirer interfered with the operation of the orchestrator causing, for instance, rebalance to fail.
+
+| https://issues.couchbase.com/browse/MB-33351[MB-33351^]
+| *Summary*: Fixed an issue where the breakpad did not catch the memcached segfault.
+
+| https://issues.couchbase.com/browse/MB-33321[MB-33321^]
+| *Summary*: Fixed an issue to prevent map corruption due to a change in `num_replicas` after an interrupted rebalance operation.
+
+| https://issues.couchbase.com/browse/MB-32840[MB-32840^]
+| *Summary*: Fixed an issue where the audit daemon asserted when rotating an empty audit log causing a crash.
+
+| https://issues.couchbase.com/browse/MB-32685[MB-32685^]
+| *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
+
+
+| https://issues.couchbase.com/browse/MB-15009[MB-15009^]
+| *Summary*: Defragment document StoredValue in addition to blobs to help cases where a large number of documents with a given key size are deleted, and then new documents are allocated with a different key size. In this case the original size class would not be able to be reused, and hence fragmentation could be high in that class.
```

Comment:
This isn't a bug-fix, it's an improvement, so I'd re-phrase to something like:

*Summary*: Improve the effectiveness of the active defragmenter to also defragment Document metadata (`StoredValue` objects) in addition to document values. This should reduce memory fragmentation for workloads which have multiple document metadata sizes (i.e. a range of key lengths).